src/seedu/addressbook/commands/FindCommand.java

```diff
@@ -5,6 +5,7 @@
 import java.util.HashSet;
 import java.util.List;
 import java.util.Set;
+import java.lang.*;
```

Comment:
Avoid batch imports.

You can change the IntelliJ settings to help prevent this.